src/Http/Http.Results/src/TypedResults.cs

```diff
@@ -486,6 +488,7 @@ public static JsonHttpResult<TValue> Json<TValue>(TValue? data, JsonSerializerCo
             LastModified = lastModified,
             EntityTag = entityTag,
             FileDownloadName = fileDownloadName,
+            EnableRangeProcessing = enableRangeProcessing,
```

Comment:
This will cause ranges to be calculated and headers set, but it doesn't actually serve the correct content. The callback doesn't know which range to server.
https://github.com/dotnet/aspnetcore/blob/419065590260e6597a994007512d03736e744da1/src/Http/Http.Results/src/PushStreamHttpResult.cs#L109-L121